packages/web/components/cards/stake-dashboard.tsx

```diff
@@ -106,16 +106,32 @@ export const StakeDashboard: React.FC<{
           validators={validators}
           usersValidatorsMap={usersValidatorsMap}
         />
-        <div className="flex h-full w-full flex-grow flex-row space-x-2">
+        <div className="flex h-full max-h-[9.375rem] w-full flex-grow flex-row space-x-2">
```

Comment:
a couple of magic numbers here in the css in this component. Are they all one offs? Why do they all have such precise decimal point rems styles?

Reply:
It may be necessary to do this to match designs, but generally rem should be preferred over px to better accommodate the user's font size setting IMO